apps/Gruntfile.js

```diff
@@ -1019,7 +1021,8 @@ describe('entry tests', () => {
         'common',
         'tutorialExplorer',
         'regionalPartnerSearch',
-        'regionalPartnerMiniContact'
+        'regionalPartnerMiniContact',
+        'donorTeacherBanner'
```

Comment:
Why do we call this `teacherDonorBanner` in some places and `donorTeacherBanner` in others?